stl/inc/ranges

```diff
@@ -29,6 +29,69 @@ namespace ranges {
     template <class _Rng>
     concept viewable_range = range<_Rng>
         && (borrowed_range<_Rng> || view<remove_cvref_t<_Rng>>);
+
+
+    // CLASS ranges::ref_view
+    // clang-format off
+    template <range _Rng>
+        requires is_object_v<_Rng>
+    class ref_view : public view_interface<ref_view<_Rng>> {
+        // clang-format on
+        void _Rvalue_poison(_Rng&);
+        void _Rvalue_poison(_Rng&&) = delete;

```

Comment:
In `class`es, we conventionally say `private` for members instead of relying on the default (unlike `struct`s where we gladly use the `public` default).

Reply:
Done